src/physics.F90

```diff
@@ -465,6 +472,12 @@ subroutine elastic_scatter(i_nuclide, rxn, E, uvw, mu_lab, wgt)
     ! Set energy and direction of particle in LAB frame
     uvw = v_n / vel
 
+    ! Because of floating-point roundoff, it may be possible for mu_lab to be
+    ! outside of the range [-1,1). In these cases, we just set mu_lab to exactly
+    ! -1 or 1
+
+    if (abs(mu_lab) > ONE) mu_lab = sign(ONE,mu_lab)
```

Comment:
Yes, actually, so I removed the one from scatter itself.  I need the ones in *_scatter because this needs to occur before rotate_angle is called, and I can't pull up the rotate_angle call to scatter() because inelastic_scatter needs rotate_angle to be called before particles are added to the secondary bank.

You are right that there is some redundant work going on here so I removed the check in scatter itself.  I checked the code and it looks like all paths are covered and I also re-ran a big problem and had no binary search errors on my mu filters, so this should be all set.